hs_core/views/resource_rest_api.py

```diff
@@ -275,6 +325,84 @@ def create(self, request, *args, **kwargs):
         return Response(data=response_data,  status=status.HTTP_201_CREATED)
 
 
+class SystemMetadataRetrieve(APIView, ResourceToListItemMixin):
+    """
+    Retrieve resource science metadata
+
+    REST URL: hsapi/sysmeta/{pk}
+    HTTP method: GET
+
+    :type pk: str
+    :param pk: id of the resource
+    :return: system metadata as JSON string
+    :rtype: str
+    :raises:
+    NotFound: return JSON format: {'detail': 'No resource was found for resource id:pk'}
+    PermissionDenied: return JSON format: {'detail': 'You do not have permission to perform this action.'}
+
+    example return JSON format for GET hsapi/sysmeta/<RESOURCE_ID>:
+
+    {
+        "resource_type": resource type,
+        "resource_title": resource title,
+        "resource_id": resource id,
+        "creator": creator user name,
+        "date_created": date resource created,
+        "date_last_updated": date resource last updated,
+        "public": True or False,
+        "bag_url": link to bag file,
+        "science_metadata_url": link to science metadata
+    }
+    """
+    @property
+    def allowed_methods(self):
+        return ['GET']
```

Comment:
no reason to make this a method if its static anyway.

`allowed_methods = ('GET',)`